app/models/wild_collection.rb

```diff
@@ -68,6 +68,17 @@ class WildCollection < ApplicationRecord
   validate :initial_holding_facility_is_valid,
            :final_facility_and_date_of_arrival_is_valid
 
+
+  def self.create_from_csv_data(attrs)
+    attrs['proximity_to_nearest_neighbor']              = attrs.delete('proximity_to_nearest_neighbor_m')
+    attrs['collection_depth']                           = attrs.delete('collection_depth_m')
+    attrs['collection_coordinates']                     = attrs.delete('collection_coodinates')
+    attrs['final_holding_facility_and_date_of_arrival'] = attrs.delete('final_holding_facility__date_of_arrival')
+    attrs['collection_date'] = Date.strptime(attrs['collection_date'], '%m/%d/%y') rescue nil
+    attrs['otc_treatment_completion_date'] = Date.strptime(attrs['otc_treatment_completion_date'], '%m/%d/%y') rescue nil
+    new(attrs)
```

Comment:
Part of the move of the CSV-to-model conversion from the jobs to the models. This is basically the preprocessing and the translate taken care of.